.circleci/config.yml

```diff
@@ -30,6 +30,20 @@ jobs:
       command: nix-shell --run hc-rust-test
       no_output_timeout: 20m
 
+ cov-kcov:
```

Comment:
Only comment I have is maybe we can change this to "Code Coverage" when it shows up? but definitely not necessarily at all

Reply:
:+1: - done